islandora_large_image.install

```diff
@@ -75,11 +75,11 @@ function _islandora_large_image_install_has_kakadu() {
 
   $out = array();
   $ret = 1;
-
-  $command = 'command -v kdu_compress >/dev/null 2>&1';
+  $kdu_compress = variable_get('islandora_kakadu_url', 'kdu_compress');
+  $command = "command -v $kdu_compress >/dev/null 2>&1";
```

Comment:
Hi, just a tiny issue here. $kdu_compress needs to be escaped for unix and windoes or made "safe" -somehow-here. I know not a big vulnerability but this can lead easily to arbitrary code execution. Maybe just via http://www.php.net/escapeshellarg. @Islandora/7-x-1-x-committers anyone with a simple to follow example here?